src/Build.UnitTests/BuildEventArgsSerialization_Tests.cs

```diff
@@ -38,13 +38,8 @@ public void RoundtripBuildStartedEventArgs()
 
             args = new BuildStartedEventArgs(
                 "M",
-                null,
-                new Dictionary<string, string>
-                {
-                    { "SampleName", "SampleValue" }
-                });
+                null);
```

Comment:
maybe keep both scenarios, having null and the dictionary are still valid cases (if the user turns on the trait)